core/src/main/java/com/linecorp/armeria/server/TransientService.java

```diff
@@ -27,4 +30,15 @@
  */
 @FunctionalInterface
 public interface TransientService<I extends Request, O extends Response> extends Service<I, O> {
+
+    /**
+     * Returns the {@link Set} of {@link TransientServiceOption}s that are enabled for this
+     * {@link TransientService}. This returns {@link Flags#transientServiceOptions()} if you didn't
+     * specify any {@link TransientServiceOption}s using
+     * {@link TransientServiceBuilder#transientServiceOptions(TransientServiceOption...)} when you create
+     * this {@link TransientService}.
+     */
+    default Set<TransientServiceOption> options() {
```

Comment:
Should we rename this to `transientServiceOptions()`, just in case a user wants to use the method name `option`, since it's fairly generic?

Reply:
No objection. 👍